src/store/modules/Logs/DeconfigurationRecordsStore.js

```diff
@@ -82,7 +82,7 @@ const DeconfigurationRecordsStore = {
                 uri: log['@odata.id'],
                 severity: Severity,
                 location: LocationCode,
-                pelID: pelId,
+                EventID: eventId,
```

Comment:
Lowercase e is preferred in the left hand side: EventID.

Reply:
Addressed